nomenclature/validation_schemas/tag_schema.yaml

```diff
@@ -32,7 +32,7 @@ definitions:
         # Attributes-dictionary (replace `<Tag>` by `value` in the `key` attribute)
         type: object
         additionalProperties:
-          type: [string, number, "null"]
+          type: [string, number, boolean, "null"]
```

Comment:
Just tested it without the `additionalProperties` tag and all tests still run through fine. To me this suggests that either this property is not required or there is a unit test missing that covers this.

Reply:
I didn't add a test, because we should have confidence that **jsonschema** works as expected with correct specifications.

Reply:
I didn't mean testing that jsonschema works, I meant testing that the schema correctly specifies the structure that we expect.

Reply:
Yes, I added these additional types because I needed them to migrate the openENTRANCE-nomenclature...

Anyway, I added some more items to the test-data to cover more cases explicitly.

Reply:
Perfect